corehq/apps/translations/models.py

```diff
@@ -21,23 +22,10 @@ def set_translation(self, lang, key, value):
     def set_translations(self, lang, translations):
         self.translations[lang] = translations
 
-    
+
 class TranslationDoc(TranslationMixin):
```

Comment:
didn't have a chance to look into that, but it seems we can probably drop it, just need to confirm